web/serve.py

```diff
@@ -20,12 +21,18 @@ class User(db.Model):
     name = db.Column(db.String(200))
 
 db.create_all()
+example_user = User(id=1, name="Philip Sterne")
```

Comment:
I also included some simple code to create example data in the db (and a route that queries the data too). 

This is not a great solution, since it unintentionally also creates the database every time. 🤷‍♂️